DeviceTests/Caboodle.DeviceTests.Shared/Permissions_Tests.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Caboodle;
+using Xunit;
+
+#if __ANDROID__
+[assembly: Android.App.UsesPermission(Android.Manifest.Permission.BatteryStats)]
+#endif
```

Comment:
Does this have to be here? Isn't it in the manifest?

Reply:
This makes sure it is in the manifest... I like doing it attributes especially for a unit test which requires it :)

Reply:
No, I mean isn't it already in the manifest? Maybe we should remove it there - or maybe it is fine for both locations?

Reply:
You know, it _is_ already there, which I thought it wasn't earlier but I believe that was due to me trying to check for the BATTERY_STATS permission as if it required user input and it was returning false because it didn't.  In any case I think it's fine in both places.